src/utilities.js

```diff
@@ -1,673 +1,693 @@
+	}
+};
+
+/**
+ * Gets a random number between min and max.
+ * The maximum is exclusive and the minimum is inclusive
+ * 
+ * @param {Number} min - the minimum
+ * @param {Number} max - the maximum 
+ */
+exports.getRand = function(min, max) {
+	min = Math.ceil(min);
+	max = Math.floor(max);
+	return Math.floor(Math.random() * (max - min)) + min; 
+};
+
+/**
+ * Gets a timestamp representing the current time.
+ * 
+ * @return {String} properly formatted timestamp
+ */
+exports.getTimestamp = function() {
+	function pad(n) {
+			return (n < 10) ? `0${n}` : n;
+	}
+
+	const time = new Date();
+	const day = c.DAYS[time.getDay()];
+	var hours = time.getHours();
+	var minutes = time.getMinutes();
+	var meridiem = 'AM';
+
+	if (hours > 12) {
+		hours -= 12;
+		meridiem = 'PM';
+	} else if (hours === 0) {
+		hours = 12;
+	}
+
+	return `${day} ${pad(hours)}:${pad(minutes)} ${meridiem}`;
+};
+
+/**
+ * Logs the response of an API request for Add Role or Move User.
+ * 
+ * @param {String} error - error returned from API request
+ * @param {Object} response - response returned from API request
+ */
+exports.log = function(error, response) {
+	if (error) {
+		c.LOG.error(`<API ERROR> ${exports.getTimestamp()}  ERROR: ${error}`);			
+	} else if (response) {
+		c.LOG.info(`<API RESPONSE> ${exports.getTimestamp()}  ${inspect(response)}`);
+	}
+};
+
+/**
+ * Builds an embed field object with name and value.
+ * 
+ * @param {String} name - the name
+ * @param {Number} value - the value
+ */
+exports.buildField = function(name, value) {
+	return {
+		name: name,
+		value: value,
+		inline: 'true'
+	};
+};
+
+/**
+ * Comparator for two field objects. Compares values.
+ * 
+ * @param {Object} a - first field
+ * @param {Object} b - second field
+ */
+exports.compareFieldValues = function(a,b) {
+	const aNum = Number(a.value);
+	const bNum = Number(b.value);
+
+	if ( aNum > bNum)
+	  return -1;
+	if (aNum < bNum)
+	  return 1;
+	return 0;
+};
+
+/**
+ * Output vote count to bot-spam channel
+ */
+exports.sendEmbedFieldsMessage = function(title, fields, userID) {
+	if (fields.length === 1 && fields[0].name === '') {
+		return;
+	}
+
+	const color = userIDToColor[userID] || 0xffff00;	
+	bot.getBotSpam().send(new Discord.RichEmbed({
+		color: color,
+		title: title,
+		fields: fields
+	}));	
+};
+
+/**
+ * Sends an embed message to bot-spam with an optional title, description, and image.
+ */
+exports.sendEmbedMessage = function(title, description, userID, image) {
+	//these are all optional parameters
+	title = title || '';
+	description = description || '';
+	image = image || '';
+	const color = userIDToColor[userID] || 0xffff00;
+	
+	bot.getBotSpam().send(new Discord.RichEmbed({
+		color: color,
+		title: title,
+		description: description,
+		image: {
+			url: image
+		} 
+	}));	
+};
+
+/**
+ * Sends an embed message to bot-spam with an optional title, description, and image.
+ */
+exports.sendEmbedMessageThumbnail = function(title, description, userID, image) {

```

Comment:
Duplicate code of above. Replace sendEmbedMessage() with this new code:
```js
exports.sendEmbedMessage = function(title, description, userID, image, thumbnail) {
	//these are all optional parameters
	title = title || '';
	description = description || '';
	image = image || '';
	const picType = thumbnail ? 'thumbnail' : 'image';
	const color = userIDToColor[userID] || 0xffff00;
	var message = {
		color: color,
		title: title,
		description: description
	};
	message[picType] = { url: image };
	bot.getBotSpam().send(new Discord.RichEmbed(message));	
};
```

call that function and pass true into last param.